### DISCUSSION TITLE
Form Builder with relation to User Email

### DISCUSSION THREAD

**Discussion by maxkarkowski** [payloadcms/payload#9927] [Q&A]
Hey, i am not quite sure if it is the right place. i want to build a Course Collection and for each course you can send a form. 

The form should always be send to the website admin but also to an organier email which is related to a single Course. The form itself is always the same for each course(name, mail, textarea) but the organizers changes for each course

**Comment by mikecebul** [comment]
You can use the Form Builder Plugin. You have two options.

1. Create a form and duplicate it for each course, updating the title and emails.
2. Use one form and handle emails with the beforeEmail property
```ts
beforeEmail: (emailsToSend, beforeChangeParams) => {
  const { data } = beforeChangeParams

  emailsToSend.forEach((email) => {
    email.cc = data.course === 'beginner' ? 'jack@example.com' : 'jan@example.com'
  })

  return emailsToSend
},
```

**Reply by maxkarkowski** [reply]
thanks for the reply. but how do i get the info of the cpurse in data? data has only the information about the form(which is a relation field in course) but has no info about the actual course
